### DISCUSSION TITLE
ODI / HSGQ DFP-34X-2C2 / DFP-34X-2C3 Firmware

### DISCUSSION THREAD

**Discussion by rapdodge** [Anime4000/RTL960x#328] [General]
I've got several firmware for DFP-34X-2C2 / DFP-34X-2C3

[ODI  -- HSGQ.zip](https://github.com/user-attachments/files/17484842/ODI.--.HSGQ.zip)

The HSGQ version I've got it from regional tech support enginner of HSGQ, and the ODI version I've got it from the sales

**Comment by Anime4000** [comment]
alright, which one is works for you?

as for Malaysia, none of this support ME 350, 370 & 373 (Huawei Proprietary OMCI) for some Huawei only network such as TIME, Maxis and Allo

for Telekom Malaysia, using multiple OLT brand from ZTE, Nokia, Huawei and Fiberhome, ME 351 is needed which only exist on Nijika Fw

for others, need feedback, as fw `220923` (SDK 1.0) works very well with PPTP UNI and VEIP UNI which can bridge both.
some are reported after fw >= `230923` (SDK 1.1) having issue to bridge VEIP UNI

I not sure why SDK 1.1 cause issue to bridge VEIP UNI, even change FwdOp not help, VEIP `0x601` and `0xe01` most common VEIP UNI having issue to bridge

**Reply by rapdodge** [reply]
> alright, which one is works for you?

Right now, I'm using the HSGQ version of `V1.1.8-240408`

The ODI version `220923` works too, but the ONU ID only 0, and with the `V1.1.8-240408`, there is ONU ID

![image](https://github.com/user-attachments/assets/c57e4dba-76fd-4be8-9cbf-2e0247a57319)

But, both are still functional well...

**Reply by Anime4000** [reply]
Alright,

I heard some ISP in Indonesia block bridging ?

I have verified ONU ID with my ISP, the stick report is wrong ID compared with OLT assigned to, as my ISP don't care ONT ID as it show an random number for GEM slots

**Reply by rapdodge** [reply]
> I heard some ISP in Indonesia block bridging ?

Yes. Most of them doesn't like to be bridged. Because reselling things...

But yea, I'm really grateful that my ISP didn't care about that thing, but they doing some shit with the ONT, everytime the ONT got restarted. The configuration will be back to routed... That's why I'm buying this ODI 🤣

**Reply by Anime4000** [reply]
I wondering what ME cause making ONT as Route Mode? the stock ONT has TR069 enabled?

**Reply by rapdodge** [reply]
> I wondering what ME cause making ONT as Route Mode? the stock ONT has TR069 enabled?

No, the ONT itself doesnt have TR069 😂 
That's why I have no clue about that...

**Reply by rapdodge** [reply]
> alright, which one is works for you?
> 
> as for Malaysia, none of this support ME 350, 370 & 373 (Huawei Proprietary OMCI) for some Huawei only network such as TIME, Maxis and Allo
> 
> for Telekom Malaysia, using multiple OLT brand from ZTE, Nokia, Huawei and Fiberhome, ME 351 is needed which only exist on Nijika Fw
> 
> for others, need feedback, as fw `220923` (SDK 1.0) works very well with PPTP UNI and VEIP UNI which can bridge both. some are reported after fw >= `230923` (SDK 1.1) having issue to bridge VEIP UNI
> 
> I not sure why SDK 1.1 cause issue to bridge VEIP UNI, even change FwdOp not help, VEIP `0x601` and `0xe01` most common VEIP UNI having issue to bridge

Btw, talking about the proprietary ME things, tripleoxygen has released the 240408 on the page

https://www.tripleoxygen.net/wiki/ont/odi/dfp-34x-2c2

**Reply by Anime4000** [reply]
that works for you?

**Reply by rapdodge** [reply]
> that works for you?

Can't test, since my ISP doesn't using the proprietary ME
